conda/cpuonly/meta.yaml

```diff
@@ -1,8 +1,10 @@
 package:
   name: cpuonly
-  version: 1.0
+  version: 2.0
 
 build:
-  track_features:
-      - cpuonly
   noarch: generic
+
+requirements:
+  run:
+    - pytorch-mutex 1.0 cpu
```

Comment:
An alternative option to `pytorch-mutex 1.0 cpu` is to use `pytorch=*=*cpu*`. However, running 
```
conda install -c nvidia -c pytorch pytorch=*=*cpu*
``` 
prefers the main channel
```
pytorch            pkgs/main/linux-64::pytorch-1.8.1-cpu_py38h60491be_0
```
So I think using `pytorch=*=*cpu*` for `cpuonly` package could break conda.

`mamba` chooses the pytorch channel
```
  + pytorch               1.9.0  py3.8_cpu_0    pytorch/linux-64        73 MB
```